app/modules/intelligence/provider/provider_service.py

```
@@ -37,263 +28,57 @@
-                description="An open-source AI company known for powerful chat and reasoning models.",
-            ),
+                id="litelm",
+                name="LiteLLM",
+                description="A provider for running open source models locally using LiteLLM.",
+            )
         ]
 
     async def set_global_ai_provider(self, user_id: str, provider: str):
-        provider = provider.lower()
-        # First check if preferences exist
+        provider = litellm_provider # Force all users to use LiteLLM
```

Ruff (0.8.2)

Local variable `provider` is assigned to but never used

Remove assignment to unused variable `provider`

(F841)